### PR TITLE
feat(msp): Replace accessKey with token

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
Replace accessKey with token

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 1]
```diff
@@ -134,7 +132,16 @@ const Configuration = () => {
   };
 
   const columns: Array<ColumnProps<CONFIGURATION.IAllKeyData>> = [
-    { title: 'accessKey', dataIndex: 'accessKey', key: 'accessKey' },
+    {
+      title: 'Token',
+      dataIndex: 'accessKey',
+      key: 'accessKey',
+      render: (_: unknown, record?: CONFIGURATION.IAllKeyData) => (
+        <Copy selector=".container-key" copyText={record?.accessKey}>
```
Comment (daskyrk) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:140]: The selector same to be useless

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 2]
```diff
@@ -257,7 +259,7 @@ const Configuration = () => {
             })
           }
           width={720}
-          title={mode === 'query' ? i18n.t('msp:accessKey details') : i18n.t('established successfully')}
+          title={mode === 'query' ? 'next version need' : i18n.t('established successfully')}
```
Comment (daskyrk) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:262]: What does this mean to user? Why not translate?

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 3]
```diff
@@ -133,55 +131,49 @@ const Configuration = () => {
...
   };
 
-  const columns: Array<ColumnProps<CONFIGURATION.IAllKeyData>> = [
-    { title: 'accessKey', dataIndex: 'accessKey', key: 'accessKey' },
+  const columns: Array<ColumnProps<CONFIGURATION.IAllTokenData>> = [
+    {
+      title: 'Token',
+      dataIndex: 'token',
+      key: 'token',
+      render: (_: unknown, record?: CONFIGURATION.IAllTokenData) =>
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:139]: use first params

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 4]
```diff
@@ -133,55 +131,49 @@ const Configuration = () => {
...
+          record &&
+          `${record?.token.substr(0, 2)}${new Array(record?.token.length - 4).join('*')}${record?.token.substr(-2)}`
+        ),
+    },
     {
       title: i18n.t('create time'),
       dataIndex: 'createdAt',
       key: 'createdAt',
-      render: (_: unknown, record?: CONFIGURATION.IAllKeyData) =>
+      render: (_: unknown, record?: CONFIGURATION.IAllTokenData) =>
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:151]: same as above

**Pull request merged**